### ISSUE TITLE
Proposal of more photos

### ISSUE THREAD

**Issue by Alexx23** [shanlh/vue-tinder#26]
Hi.

I think that you could add more photos in the current object, that is, that you can see more photos in carousel format for the object you are currently viewing.

I do not know if I have explained myself properly, but this is implemented by Tinder and other applications.

Thanks.

**Comment by shanlh** [comment]
Yes, it's supported. But it's need you code in the default slot.
It’s a bit complicated as a demo, so I didn't make it.

**Comment by Alexx23** [comment]
Thanks for your reply.

Is there not an example that I can see to know how to do it?
